.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: CI
+
+on:
+  push:
```

Comment:
Please also add regular rerun once very month or so
As otherwise we cannot automatically detect problems without opening a new PR